apps/authentication/views.py

```diff
@@ -83,9 +85,12 @@ def register(request):
 
                 # Create the registration token
                 token = uuid.uuid4().hex
-                rt = RegisterToken(user=user, email=email.email, token=token)
-                rt.save()
-
+                try:
+                    rt = RegisterToken(user=user, email=email.email, token=token)
+                    rt.save()
+                    log.info('Successfully registered token for %s' % request.user)
+                except:
+                    log.error('Failed to register token for %s' % request.user)
```

Comment:
This way, if the code fails it'll just keep going. What needs to happen, is to make the program generate a _new_ code, so it actually makes sense to continue. 

We need the program to create a valid code.

This functionality could override the `create()`-function of the object, or create a custom function which we use instead of the default `create()`.